data/json/recipes/armor/legs.json

```diff
@@ -282,6 +282,20 @@
     "components": [ [ [ "paper", 60 ] ], [ [ "duct_tape", 10 ] ] ],
     "flags": [ "BLIND_HARD" ]
   },
+  {
+    "result": "legguard_scrap",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_LEGS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "20 m",
```

Comment:
Just twenty minutes to tie up 60 pieces of metal scraps?

Reply:
It's been lengthened